README.md

```diff
@@ -1,20 +1,106 @@
+
+The DPS also supports a set of custom proxy resources for token vaults and staking/delegating resources. This allows the
+DPS to track account balances for locked, staked and delegated tokens for accounts which deploy these custom proxy
+resources.
+
 ## Road Map
 
-- bootstrap from checkpoint
-  - (is the first update the root checkpoint?)
-    - check Flow Go code
-- add proper logging
-- clean up component interfaces
+| Milestone |                  Description                  | Past Spork State | Past Spork Events | Live Spork State | Live Spork Events | Raw API | Ledger API | Rosetta API | Liquid Balance | Locked Balance | Staked Balance | Delegated Balance | State Verification | State Proofs |
+|:---------:|:---------------------------------------------:|:----------------:|-------------------|------------------|-------------------|---------|------------|-------------|----------------|----------------|----------------|-------------------|--------------------|--------------|
+|    P.1    |        Past Spork support for registers       |         X        |                   |                  |                   |    X    |      X     |             |        X       |                |                |                   |                    |              |
+|    P.2    |         Past Spork support with events        |         X        |         X         |                  |                   |    X    |      X     |             |        X       |                |                |                   |                    |              |
+|    R.1    |    Rosetta API support for default balance    |         X        |         X         |                  |                   |    X    |      X     |      X      |        X       |                |                |                   |                    |              |
+|    L.1    |        Live Spork support for registers       |         X        |         X         |         X        |                   |    X    |      X     |      X      |        X       |                |                |                   |                    |              |
+|    L.2    |         Live Spork support with events        |         X        |         X         |         X        |         X         |    X    |      X     |      X      |        X       |                |                |                   |                    |              |
+|    R.2    | Rosetta API support with sub-account balances |         X        |         X         |         X        |         X         |    X    |      X     |      X      |        X       |        X       |        X       |         X         |                    |              |
+|    C.1    |   Cryptographic Verification of local state   |         X        |         X         |         X        |         X         |    X    |      X     |      X      |        X       |        X       |        X       |         X         |          X         |              |
```

Comment:
I am wondering what is included in this Milestone (?) 

* The DPS receives registers updates (either from the WAL or per live update). The data source here are Execution Nodes (either directly or indirectly).  Furthermore, the seal (included in the block) contains a state root hash. 

Is this Milestone to confirm that the updated registers have indeed the same root hash as stated in the seal? If this is the case, would it be possible to to move this up, e.g. have a milestone "Cryptographic Verification of Past Spork register values" (right after `P.1`)?  From my perspective, verifying that the local register values actually are consistent with the state root hash (published in the seal) is an important sanity check.

Reply:
From what I understand, this milestone is indeed to confirm that the updated registers have the same root hash as stated in the seal. I am fine with moving it up. Sounds good to you @awfm9 ?

Reply:
We ended up removing the `C.1` milestone since it is already implemented as part of `P.1` in the end. We then split the original `C.2` into two milestones: one is cryptographic proofs for registers and the other for events.

Reply:
I realized that replaying the state actually requires validating the trie against the incoming updates at each point, so we already validate the state anyway. However, event verification is not currently supported, as there is no basis to do so yet.